### DISCUSSION TITLE
More than 2 words in camera name?  How to refresh camera list?

### DISCUSSION THREAD

**Discussion by hiroo916** [mrlt8/docker-wyze-bridge#129] [Q&A]
I've got the wyze bridge docker working and can access 2 out of 5 of my cameras.  The 3 that don't work show an error in the docker CLI of (for example) "ERR: no one is publishing to path 'edu-front-door'.  However, at startup, it shows  "[path edu-front-door] created" so it seems like it is reading the camera name correctly from the account. 

The only thing different I can notice about the 3 that don't work vs the 2 that do work is the the working cameras have two word names (example: back-door) vs the 3 that don't work have 3 or more words separated by spaces. 

Is there a problem with multi-word names?

I also tried change the names of the non-working cameras to shorten to two words, but the docker doesn't seem to refresh the cameras list. I assume this is because it is reading from the data cached at first run.  Is there a way to flush the cache and refresh from the wyze account without rebuilding the whole docker? (sorry, I'm new to dockers)

**Comment by mrlt8** [comment]
Spaces shouldn't be an issue. Are they all on the same firmware? 

You can force the container to pull fresh data from the API with the `FRESH_DATA=True` env variable.

`-e FRESH_DATA=True` if using docker run or add  it somewhere in under environment if using docker-compose:
```yaml
        environment:
            - WYZE_EMAIL=user@email.com
            - WYZE_PASSWORD=pass
            - FRESH_DATA=True
```
